README.md

```diff
@@ -129,6 +127,36 @@ Each project's pull request build pipeline should ensure no lint errors can be c
 
 Typically steps 1-3 will happen in a single pull request (or a few in quick succession) while step 4 will be split across many subsequent submissions as time permits.
 
+### Evaluate project-specific rule groups
+
+Several sets of rules may be enabled based on requirements of a given project. By default the following sets of rules are in an inert / disabled state, but should be evaluated for your integration.
+
+Text search for the tag associated with a specific rule group in the repository to find the related rules. If enabling a rule group, the rules should be toggled from `'off'` to `'error'` unless the rule comment says otherwise.
+
+#### Application prefix
+
+Tag: `[application-prefix]`
+
+Prefixes are generally added to named objects such as the selector for Components in Angular applications. Projects should consider enabling this rule group so that names can be consistently prefixed making them easier to share between applications and to minimize the chance of conflicts when using shared libraries.
+
+#### Strict null checks
+
+Tag: `[strict-null-checks]`
+
+When `strictNullChecks` are enabled the values `null` and `undefined` are treated as distinct types by the compiler. For example, with `strictNullChecks` enabled, the value `null` could not be directly assigned to a binding of a `Cat` object, ie `const cat: Cat = null` would be a compile error. The `null` value is a distinct type and the binding would have to explicitly state that it can have a `null` value, ie `const cat: Cat | null = null;`.
+
+`strictNullChecks` are a powerful tool for code correctness and give us a way to avoid ["The Billion Dollar Mistake"](https://www.infoq.com/presentations/Null-References-The-Billion-Dollar-Mistake-Tony-Hoare/). However, it can be impractical to retrofit `strictNullChecks` configuration into an existing application and requires expanding your mental model for software development for use in new applications.
+
+As such, `strictNullChecks` are not recommended by default in order to prevent overhead of rule adoption for existing applications.
+
+However, we encourage new applications to leverage `strictNullChecks` for development. Enabling `strictNullChecks` is the TypeScript compiler recommendation and it is enabled by default in new Angular applications.
+
+#### Accessibility
+
+Tag: `[accessibility]`
```

Comment:
```suggestion
Tag: [`[accessibility]`](https://github.com/ni/javascript-styleguide/search?q=%5Baccessibility%5D)
```

Reply:
I had the exact same thought but with also including `&type=code` in the url parameters. My concern is that GitHub search doesn't include punctuation. See the current [accessibility](https://github.com/ni/javascript-styleguide/search?q=%5Baccessibility%5D) search. The github.dev vscode search does but I couldn't figure out a way to encode a search in URL parameters.

I was on the fence but if you confirm a vote for still including the url (with `&type=code` added) then I'll jump on board too.

Reply:
Yeah, it's not perfect but I figure there aren't likely to be too many spurious results so it's still valuable.